src/libp2p_connection.erl

```diff
@@ -141,6 +141,7 @@ mk_async_sender(Handler, Connection) ->
                      end
              end,
     fun() ->
+            erlang:put(async_sender_for, Parent),
```

Comment:
I'm missing where this is used?

Reply:
Identifying what these processes are for and what connectioon they relate to